pkg/controller/core.oam.dev/v1beta1/application/generator.go

```diff
@@ -252,6 +252,7 @@ func convertStepProperties(step *workflowv1alpha1.WorkflowStep, app *v1beta1.App
 	o := struct {
 		Component string `json:"component"`
 		Cluster   string `json:"cluster"`
+		Namespace string `json:"namespace"`
```

Comment:
please use the application namespace if empty.

Reply:
have done @wonderflow